Storage/ThirdParty/SwiftData.swift

```diff
@@ -157,20 +157,32 @@ open class SwiftData {
     func transaction(synchronous: Bool=true, transactionClosure: @escaping (_ db: SQLiteDBConnection) -> Bool) -> NSError? {
         return withConnection(SwiftData.Flags.readWriteCreate, synchronous: synchronous) { db in
             if let err = db.executeChange("BEGIN EXCLUSIVE") {
-                log.warning("BEGIN EXCLUSIVE failed.")
+                log.error("BEGIN EXCLUSIVE failed. \(err)")
+                SentryIntegration.shared.sendWithStacktrace(message: "BEGIN EXCLUSIVE failed. \(err)", tag: "SwiftData", severity: .error)
```

Comment:
It should give us `localizedDescription`. We can probably also log the `code` just in case.